.idea/dictionaries/kevint.xml

```diff
@@ -1,6 +1,9 @@
 <component name="ProjectDictionaryState">
   <dictionary name="kevint">
```

Comment:
Is that your personal dictionary (or where does `kevin?  come from)?

Reply:
[this confuses everyone](https://keturn.net/blog/posts/2020/05/git-add-dot-idea-share-intellij/#idea/dictionaries) but they've been resistant to changing their implementation